typhon/plots/common.py

```diff
@@ -2,15 +2,21 @@
 
 """Utility functions related to plotting.
 """
+
+import pathlib
+import lzma
 import glob
 import itertools
 import os
 import string
+import pickle
+import logging
 
 import matplotlib.pyplot as plt
 import numpy as np
 from typhon import constants
 
+LOG = logging.getLogger(__name__)
```

Comment:
Variable names should follow the [PEP8](https://www.python.org/dev/peps/pep-0008/#global-variable-names) convention. Use `logger` to be consistent with the rest of typhon.

Reply:
Changed `LOG` to `logger` in latest commit.